README.md

```diff
@@ -177,6 +177,22 @@ Setting `volumes` will mount volumes on the host machine to designated paths on
 
 Setting `hosts` will update the hosts configuration for the container. Entries should use the format `<HOST_NAME>:<ADDRESS>`
 
+## Service Stop Time (`stopTimeSecs <integer>`)
+
+The standard procedure for stopping a Docker container is via the `stop` command which sends `SIGTERM` and allows a grace period (default: `10s`) for the container to exit on its own.

```

Comment:
Zuko is the english nerd, but should this be?:
```diff
-procedure for stopping a Docker container is via the `stop` command
+procedure for stopping a Docker container is the `stop` command
```

Technically the default is `10` (without the `s`).